docs/api/browser-window.md

```diff
@@ -263,6 +263,8 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       When node integration is turned off, the preload script can reintroduce
       Node global symbols back to the global scope. See example
       [here](process.md#event-loaded).
+    * `inheritPreload` Boolean (optional) - Whether the preload script is inherited
```

Comment:
I thought we were supposed to add a more generic mechanism, like a callback that can modify webPreferences and add the preload path to the parent window.

Reply:
@tarruda yes, but I didn't have time yet to investigate how to do it :(